bower.json

```diff
@@ -92,6 +93,12 @@
     "marked": {
       "js": "lib/marked.js"
     },
+    "mixpanel": {
+      "js": [
+        "mixpanel-jslib-snippet.js",
+        "mixpanel.min.js"
```

Comment:
we could include the not minified version.. as we are minifying them anyway in the end..

Reply:
or maybe not.. I can see that is used in a different way.. :|

Reply:
Yeah, it's required by Mixpanel that it can load the full-fledged script from it's snippet script: https://github.com/mixpanel/mixpanel-js#using-bower-to-load-the-entire-library